tests/log_fidelity/test_bgp_shutdown.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import pytest
+
+from tests.common.plugins.loganalyzer.loganalyzer import LogAnalyzer, LogAnalyzerError
+
+logger=logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('any')
+]
+
+def check_syslog(duthost, prefix, trigger_action, expected_log, restore_action):
+    loganalyzer = LogAnalyzer(ansible_host=duthost, marker_prefix=prefix)
+    loganalyzer.expect_regex=[expected_log]
+
+    try:
+        marker=loganalyzer.init()
+        duthost.command(trigger_action)
+        logger.info("Check for expected log {} in syslog".format(expected_log))
+        loganalyzer.analyze(marker)
+
+    except LogAnalyzerError as err:
+        logger.error("Unable to find expected log in syslog")
+        raise error
+
+    finally:
+        duthost.command(restore_action)
+
+def test_bgp_shutdown(duthosts, rand_one_dut_hostname):
+    duthost=duthosts[rand_one_dut_hostname]
+
+    BGP_DOWN_EXPECTED_LOG_MESSAGE = "admin state is set to 'down'"
```

Comment:
```suggestion
    BGP_DOWN_EXPECTED_LOG_MESSAGE = ".*admin state is set to 'down'.*"
```
I _think_ you need to add the `.*` to capture everything but I am not 100% sure. Try it out!

Reply:
It seems like loganalyzer takes in and checks the regex without needing .* to specify there's extra characters in front or after this expected string. The test runs with fine with just the expected log only.